revampedClient/components/cards/Card.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../../assets/components/_card.scss';
+
+/**
+ * @description - This component displays a reusable card
+ *
+ * @param {object} props { children, style }
+ *
+ * @returns JSX
+ */
+const Card = (props) => {
+  const {
+    children, style,
+  } = props;
+
+  return (
+    <div
+      className="card"
+      style={style}
+    >
+      {children}
+    </div>
+  );
+};
+
+Card.propTypes = {
+  children: PropTypes.string.isRequired,
```

Comment:
I think children should have `PropTypes.node.isRequired`

Reply:
lol I didn't even know there was soemthing like that. thanks